kahuna/public/js/archiver/archiver.js

```diff
@@ -0,0 +1,42 @@
+import angular from 'angular';
+
+import template from './archiver.html!text';
+
+export const archiver = angular.module('gr.archiver', []);
+
+archiver.controller('grArchiverCtrl', function() {
+
+    var ctrl = this;
+    ctrl.count = 0;
+    ctrl.notArchivedCount = 0;
+    ctrl.archivedCount = 0;
+    ctrl.hasMixed = false;
+    ctrl.archiving = false;
+
+    ctrl.service.count$.subscribe(i => ctrl.count = i);
+    ctrl.service.archivedCount$.subscribe(i => ctrl.archivedCount = i);
+    ctrl.service.notArchivedCount$.subscribe(i => ctrl.notArchivedCount = i);
+    ctrl.service.hasMixed$.subscribe(hasMixed => ctrl.hasMixed = hasMixed);
```

Comment:
doesn't subscribe fire with the initial value, so you don't have to initialise above?